app/src/main/res/layout/fragment_amortisationsrechner.xml

```diff
@@ -4,20 +4,260 @@
+                    android:layout_gravity="center_horizontal"
+                    android:text="@string/amort_text_beschreibung"
+                    android:textAlignment="center"
+                    android:textSize="20sp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <TextView
+                    android:id="@+id/amort_text_verbrauch_akt"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amort_margin_top"
+                    android:text="@string/amort_text_verbrauch_akt"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@+id/text_amortisationsrechner"
+                    app:layout_constraintTop_toBottomOf="@+id/text_amortisationsrechner" />
+
+                <EditText
+                    android:id="@+id/amort_edit_verbrauch_akt"
+                    android:layout_width="@dimen/amort_editNumbers"
+                    android:layout_height="wrap_content"
+                    android:inputType="numberDecimal"
+                    android:textSize="@dimen/content_text_big"
+                    app:layout_constraintStart_toStartOf="@+id/amort_text_verbrauch_akt"
+                    app:layout_constraintTop_toBottomOf="@+id/amort_text_verbrauch_akt" />
+
+                <TextView
+                    android:id="@+id/amort_text_verbrauch_akt_kwh"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amort_edit_measure_margin_top"
+                    android:text="@string/text_kWh"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amort_edit_verbrauch_akt"
+                    app:layout_constraintTop_toTopOf="@+id/amort_edit_verbrauch_akt" />
+
+                <TextView
+                    android:id="@+id/amort_text_verbrauch_neu_kwh"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amort_edit_measure_margin_top"
+                    android:text="@string/text_kWh"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amort_edit_verbrauch_neu"
+                    app:layout_constraintTop_toTopOf="@+id/amort_edit_verbrauch_neu" />
+
+                <EditText
+                    android:id="@+id/amort_edit_verbrauch_neu"
+                    android:layout_width="@dimen/amort_editNumbers"
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in next commit.